.ci/travis/travis.sh

```diff
@@ -461,6 +461,36 @@ no-error-pmd)
   removeFolderWithProtectedFiles pmd
   ;;
 
+no-violation-test-josm)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.6.0:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  mkdir -p .ci-temp
+  cd .ci-temp
+  ANT_VERSION=1.10.7
+  wget -q http://www-us.apache.org/dist//ant/binaries/apache-ant-${ANT_VERSION}-bin.zip -O tmp.zip
```

Comment:
Can travis not supply us with ant? https://docs.travis-ci.com/user/languages/java/#projects-using-ant
Requiring an install like this may conflict with local users and prevent them from running this piece of the script locally.

Reply:
Travis use https://travis-ci.org/github/checkstyle/checkstyle/jobs/686213037#L128
```
ant version
Apache Ant(TM) version 1.9.6 compiled on July 20 2018
```

Reply:
If ant is supplied in travis execution, why do we need other version? Does this project require a specific version of ant?

Reply:
@don-vip, please reply.

Reply:
Again, I didn't wrote this code, you did. This PR is working for several days, I spent a lot of time to help you to finish what you started but I don't have time anymore to work on never-ending feedback on cosmetic changes.

Reply:
unfortunately I cannot remember all code, I have 1K PRs where all need assistance to merge it.
Quality of code is not disputable item in our project, we are not hosting of some code, code need to be finished and do not make problems to other, all visible points must be resolved. It is hard .... but it is definition of low threshold for new contributors and ease of maintainability.